internal/commands/pull/inputs.go

```diff
@@ -109,3 +118,46 @@ func (i *inputs) resolveRemoteApp(ui terminal.UI, clients cli.Clients) (realm.Ap
 
 	return app, nil
 }
+
+func (i *inputs) resolveTemplate(ui terminal.UI, realmClient realm.Client, groupID, appID string) (*zip.Reader, []string,  error) {
+	if i.TemplateID == "" {
+		return nil, nil, nil
+	}
+
+	var paths []string
```

Comment:
check me on this one, but I think you could initialize this with `paths := []string{frontendPath}`, and still overwrite it with `AskOne` (and ultimately avoid the `else` down below)